packages/next/src/server/app-render/action-handler.ts

```diff
@@ -463,8 +463,15 @@ export async function handleAction({
 
         if (isMultipartAction) {
           if (isFetchAction) {
+            const readableLimit = serverActions?.bodySizeLimit ?? '1 MB'
+            const limit = require('next/dist/compiled/bytes').parse(
+              readableLimit
+            )
             const busboy = require('busboy')
-            const bb = busboy({ headers: req.headers })
+            const bb = busboy({
+              headers: req.headers,
+              limits: { fieldSize: limit },
```

Comment:
Unsure what to do here since `busboy` is pr. field whereas `next` is for the whole body.